bukkit/rpk-permissions-bukkit/src/main/kotlin/com/rpkit/permissions/bukkit/command/group/GroupCommand.kt

```diff
@@ -30,13 +30,15 @@ class GroupCommand(private val plugin: RPKPermissionsBukkit) : CommandExecutor {
     private val groupAddCommand = GroupAddCommand(plugin)
     private val groupRemoveCommand = GroupRemoveCommand(plugin)
     private val groupListCommand = GroupListCommand(plugin)
+    private val permissionGroupListCommand = PermissionGroupListCommand(plugin)
```

Comment:
Match the pattern followed by the other commands - i.e. GroupListCommand instead of PermissionGroupListCommand
```suggestion
    private val groupListCommand = GroupListCommand(plugin)
```